e2e/backup_test.go

```diff
@@ -33,13 +36,18 @@ var _ = Context("backup", func() {
 	var restorePoint time.Time
 
 	It("should create a bucket", func() {
-		kubectlSafe(nil, "run", "--command", "make-bucket", "--image=moco-backup:dev", "--",
-			"s3cmd", "--host=minio.default.svc:9000", "--host-bucket=minio.default.svc:9000", "--no-ssl",
-			"--access_key=minioadmin", "--secret_key=minioadmin", "mb", "s3://moco")
+		kubectlSafe([]byte(makeBucketYAML), "apply", "-f", "-")
```

Comment:
This `apply` creates a Job.
It would be nice to confirm the successful completion of that Job.

Reply:
fixed at https://github.com/cybozu-go/moco/pull/580/commits/84c694f59a5c8d24f40db63b63f4292dd512cb8f